infra/bitcoin/run.sh

```diff
@@ -9,9 +9,10 @@ sleep 20
 # Print setup
 echo "BITCOIN_ADDRESS=$ADDRESS"
 
-# Create wallet
 /app/bin/bitcoin-cli createwallet "testwallet"
 
+sleep 10
```

Comment:
Is the sleep necessary?

Reply:
No it's not, that just crept in from debugging